### PR TITLE
Feature/user taskboard

### PR DESCRIPTION
**Pull request by thisdarktao**
This adds the feature as described in #109.

### CODE REVIEW

File: static/scss/_variables.scss [hunk 1]
```diff
@@ -68,7 +68,8 @@ $table-hover-color:             rgb(254, 249, 232);
 // z-indexes
 $z-layers: (
   "top": 11,
-  "tabs": 10
+  "tabs": 10,
+  "modal": 999
```
Comment (stuartaccent) [inline static/scss/_variables.scss]: 999 is def a higher number than 11 :)

**Comment by stuartaccent** [comment]
B4 it merges it will need to run `npm run build:prod` and commit the js

File: static/scss/_variables.scss (hunk 1 above)
Comment (thisdarktao) [reply static/scss/_variables.scss]: I guess I thought "top" here referred to a page element, not the topmost layer. I can change this if you like? 😍

**Comment by thisdarktao** [comment]
Just to address your questions one by one:

- What should the order be when first adding a task assignee? looks to be the default. when a task is added it works out the top order for the client/column i think (this should prob do the same i guess) **Since the order comes from the assignee itself and not the task, the order will be the default for that particular assignee.**
- Should it be top / bottom? 
- What happens when i add an assignee to multiple tasks / multiple clients? I guess the order should initially somehow reflect the priority across all tasks for you on all clients? **The ordering will start out as the default, and will gain an actual order based on the position of the tasks on the personal taskboard as they are prioritised.**
- What if i move an existing task from the bottom of todo on the client to the top. Should it move pos on the personal task board so I know its higher than i thought it was? **It's not possible to do this, since the task order is exclusive to the job. The original WIP system had a contiguous order for all tasks, regardless of the job. Ideally if this task is now top priority, it should be given a target date.**
- Whats the order of a task when moved on a filtered taskboard? Where should it appear when i unfilter it? **This is actually something I hadn't considered, but is a happy side effect of being able to filter. Perhaps the cards shouldn't be moveable while filtering as this would prevent a lot of weird edge cases?**

Essentially, what's been explicitly asked for in the past can't be done because of the way the ordering was implemented. This PR introduces a personal task board with simple ordering top / bottom based on what the user should work on next, not necessarily the most important task from the job.

If tasks on the m-TAG taskboard are prioritised top / bottom, but I am working on a Signmaster task, I do not want to see that m-TAG task at the top of my personal task board. I want to see tasks that are a priority for _me_ to work on right away.

File: static/scss/_variables.scss (hunk 1 above)
Comment (stuartaccent) [reply static/scss/_variables.scss]: I don’t care where it is or what the value is I meant 12 would have done lol

**Comment by stuartaccent** [comment]
All good then

**Pull request merged**